src/esp/gfx/Simulator.cpp

```diff
@@ -346,6 +346,13 @@ Magnum::Quaternion Simulator::getRotation(const int objectID,
   return Magnum::Quaternion();
 }
 
+bool Simulator::contactTest(const int objectID, const int sceneID) {
```

Comment:
Should we expose the physics manager instead of forwarding? It would simplify Simulator and avoid having to add forwarding code every time a new API is added.

Reply:
We've talked about this in the past. At that time the motivation was:
1) make C++ the primary API (implement functions in Simulator class in C++)
2) make physics functions callable from sim.function() instead of needing sim.physicsManager.function() for ease of use.

Reply:
For 2. you could grab a physicsManager reference and just call physicsManager.function(). But sounds reasonable to have everything go through Simulator.

Non-blocking: it would be nice to not have all the forwarding logic in Simulator.cpp. Maybe via multiple inheritance or an include.